src/app/checks/[share_token]/practice/page.tsx

```
@@ -0,0 +1,34 @@
+  }
+
+  return (
+    <PracticeStoreProvider initialStoreProps={{ questions: check.questions }}>
+      <PageHeading title='Practice' />
+
+      <div className='mx-auto'>
+        <div className='mb-8 flex gap-12'>
+          {check.questions.map((q, i) => (
+            <div key={q.id} className='relative w-fit'>
+              <span className='absolute right-0 bottom-2 left-0 text-center text-sm text-neutral-300'>{i + 1}</span>
+              <div className='h-2 w-16 rounded-xl bg-neutral-400' children='' />
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)